templates/data/kyma-integration-data.yaml

```
@@ -651,6 +668,22 @@ templates:
                     - "vm_job_template_k3d"
                     - "vm_job_labels_template"
                     - "extra_refs_kyma-local"
+              - jobConfig:
+                  name: "post-main-kyma-integration-k3d-telemetry"
```

post-main jobs also need `run_if_changed` field, otherwise they'll run on all merged PRs
```suggestion
                  name: "post-main-kyma-integration-k3d-telemetry"
                  run_if_changed: '^components/telemetry-operator/|^resources/telemetry/'
```